xrpl4j-core/src/test/java/org/xrpl/xrpl4j/codec/binary/BinarySerializationTests.java

```diff
@@ -134,9 +136,10 @@ public void serializeAccountDelete() throws JsonProcessingException {
       .sequence(UnsignedInteger.valueOf(2470665))
       .destination(Address.of("rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe"))
       .destinationTag(UnsignedInteger.valueOf(13))
+      .networkId(NetworkId.of(UnsignedInteger.MAX_VALUE))
```

Comment:
Let's add one test that is explicitly indicates (via its name) that the test is testing the absence of a network id on a particular transaction's binary encoding.